examples/task-frequency-job-execution/task_frequency_job_execution.bal

```diff
@@ -0,0 +1,43 @@
+import ballerina/io;
+import ballerina/task;
+import ballerina/time;
+
+// Creating a job to be executed by the scheduler.
+class Job {
+
+    *task:Job;
+    int i = 1;
+
+    // Get executed by the scheduler when the scheduled trigger fires.
+    public function execute() {
+        self.i += 1;
+        io:println("MyCounter: ", self.i);
+    }
+
+    isolated function init(int i) {
+        self.i = i;
+    }
+}
+
+public function main() returns error? {
+
+    // Get the current time.
+    time:Utc currentUtc = time:utcNow();
+    // Increase the time by three seconds.
```

Comment:
From a task module perspective, this means we are delaying the task by 03 seconds. shall we add it in the comment